### PR TITLE
Added Store selection option specific to EE

### PR DESCRIPTION
**Pull request by pawankparmar**
## Purpose of this pull request

This pull request (PR) fix for issue #248 
Made **Store selection option specific to EE**

If Magento Commerce, please specify:

- [ ] Commerce only?

## Affected documentation pages

- https://docs.magento.com/user-guide/system/permissions-user-roles.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/system/permissions-user-roles.md [hunk 1]
```diff
@@ -31,14 +31,14 @@ To give someone restricted access to the Admin, the first step is to create a ro
     - `All`
     - `Custom`
 
-    If set to `Custom` for a multisite installation, select the checkbox of the website and store where the role is to be used.
+    {:.ee-only} If set to `Custom` for a multisite installation, select the checkbox of the website and store where the role is to be used.
```
Comment (leslietilling) [inline src/system/permissions-user-roles.md:34]: That tag works on line items but not paragraphs (something we should add to the theme). In this case, we will have to use a span:

```suggestion
<span class="ee-only"></span>If set to `Custom` for a multisite installation, select the checkbox of the website and store where the role is to be used.
```

File: src/system/permissions-user-roles.md [hunk 2]
```diff
@@ -31,14 +31,14 @@ To give someone restricted access to the Admin, the first step is to create a ro
     - `All`
     - `Custom`
 
-    If set to `Custom` for a multisite installation, select the checkbox of the website and store where the role is to be used.
+<span class="ee-only"></span>If set to `Custom` for a multisite installation, select the checkbox of the website and store where the role is to be used.
```
Comment (leslietilling) [inline src/system/permissions-user-roles.md:34]:
```suggestion
   <span class="ee-only"></span>If set to `Custom` for a multisite installation, select the checkbox of the website and store where the role is to be used.
```

**Comment by leslietilling** [comment]
running tests

**Pull request merged**